bin/master

```diff
@@ -70,7 +70,14 @@ do
     # Achieve the above by
     # 1. remember the last scheduled commit
     # 2. schedule new jobs from the last scheduled commit
-    echo "bin/gauge $pr $commit && bin/deploy" > "$PROG_HOME/jobs/work-merge-$pr.job"
+    job="$PROG_HOME/jobs/work-merge-$pr.job"
+    test_sbt=$((RANDOM % 100 < 10))
```

Comment:
What is this for? Maybe add come comment.

Reply:
This is to only test `dotty-sbt` every 10 PRs. I'll add a comment.

Reply:
Ok